recipes/unidock/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "unidock" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dptech-corp/Uni-Dock/archive/refs/tags/{{ version }}.tar.gz
+  sha256: d8147e596079632221ef3c692cb1f1e6fc6e9a1e9303d2bcd44afe1508557af0
+
+build:
+  skip: true  # [not linux]
+  skip: true  # [cuda_compiler_version == "None"]
+  skip: true  # [cuda_compiler_version == "11.2"]
+  script: |
+    cd unidock
+    mkdir conda_build && cd conda_build
+    cmake ${CMAKE_ARGS} ${SRC_DIR}/unidock
+    make unidock -j$CPU_COUNT
+    make install
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake
+    - make
+  host:
+    - libboost-devel
+    - libcurand-dev  # [(cuda_compiler_version or "").startswith("12")]
+    - libgomp  # [linux]
```

Comment:
As `libgomp` uses [`run_exports/strong`]( https://github.com/conda-forge/ctng-compilers-feedstock/blob/5f4387542975181d20803946cb4d2c2061835ccc/recipe/meta.yaml#L454-L456 ), adding it to `requirements/build` is sufficient. It will then be automatically added to `requirements/host` and `requirements/run`

```suggestion
```